server/telemetry.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+const (
+	telemetryOauthModeJWT   = "JWT"
+	telemetryOauthModeOauth = "Oauth"
+
+	telemetryStartSourceWebapp  = "webapp"
+	telemetryStartSourceCommand = "command"
+)
+
+func (p *Plugin) trackConnect(userID string) {
+	p.tracker.TrackUserEvent("connect", userID, map[string]interface{}{})
+}
+
+func (p *Plugin) trackDisconnect(userID string) {
+	p.tracker.TrackUserEvent("disconnect", userID, map[string]interface{}{})
+}
+
+func (p *Plugin) trackOAuthModeChange(method string) {
+	p.tracker.TrackEvent("oauth_mode_change", map[string]interface{}{
+		"method": method,
+	})
+}
+
+func (p *Plugin) trackMeetingStart(userID, source string) {
+	p.tracker.TrackUserEvent("start_meeting", userID, map[string]interface{}{
```

Comment:
Looks like there is a lint error for not checking the returned error from these calls